docs/.vitepress/config.ts

```diff
@@ -0,0 +1,53 @@
+import { defineConfig } from "vitepress";
+import { rules } from "./rulesForSidebar";
+import { version } from "../../package.json";
+
+// https://vitepress.dev/reference/site-config
+export default defineConfig({
+  title: "vuejs-accessibility",
```

Comment:
"eslint-plugin-vuejs-accessibility" was too long, it overflowed from the sidebar:

![image](https://github.com/vue-a11y/eslint-plugin-vuejs-accessibility/assets/31937175/b4548673-cd3a-4341-97f2-bf1624c06eee)